src/OsmMom.sol

```diff
@@ -75,23 +71,39 @@ contract OsmMom {
 
     mapping (bytes32 => address) public osms;
 
-    constructor() public {
+    constructor(address vat_) {
+        vat = vat_;
         owner = msg.sender;
+        emit SetOwner(msg.sender);
+    }
+
+    function file(bytes32 what, address data) external onlyOwner {
+        if (what == "autoLine") {
+            autoLine = data;
+        } else {
```

Comment:
Neat - can revert on this line (dss style).

Reply:
Done!